src/amplitude/client.ts

```diff
@@ -1,9 +1,11 @@
 import amplitude from 'amplitude-js';
+import { EventData } from './events';
 
-function initializeAmplitudeClient() {
-    const apiKey = window.location.hostname === 'arbeidsgiver.nav.no'
-        ? '3a6fe32c3457e77ce81c356bb14ca886'
-        : '55477baea93c5227d8c0f6b813653615';
+function initializeAmplitudeClient(): AnalyticsClient {
+    const apiKey =
+        window.location.hostname === 'arbeidsgiver.nav.no'
+            ? '3a6fe32c3457e77ce81c356bb14ca886'
+            : '55477baea93c5227d8c0f6b813653615';
```

Comment:
Gjør litt vondt hver gang jeg ser dette :(

Reply:
Burde vi egentlig hatt amplitude-logging direkte i klient-koden? Kunne ikke bare ia-tjenester-metrikker også logget til amplitude fra serveren? Nå eksposer vi jo apinøklene direkte i kildekoden som hvem som helst kan fange opp:)

Reply:
Kanskje kunne vi lagret de som miljøvariabler ? `local` `dev``prod` ?

Reply:
Det gjør jo egentlig ingen forskjell fordi folk kan jo fremdeles hente de ut fra kildekoden

Reply:
Med mindre det er for å ikke ha apinøkkelen direkte i src så skjønner jeg det litt, men stemmer mer for å rydde opp i det ved å la serveren håndtere logg-kallene :)

Reply:
Det er en del informasjon som `amplitude.js` (amplitude client-lib) gjør for oss fra front-end som f.eks å håndtere `device_id`, `session_id`, `event_id`... osv)
Vi kunne endre konfig til `amplitude.js` så kallet blir sendt til vår frackend (server.js) og da sendt videre til amplitude.nav.no (som allerede er en proxy). Vi må passe godt på at ingen header blir tapt i denne reisen.
Alternative er å lagre `apiKey` som en secret. Den blir da litt mindre lett tilgjengelig.

Reply:
I teorien er det ikke en api-nøkkel, men en nøkkel til selve bucketen vi ønsker at dataen skal vises. Da er det litt bedre, selv om vi også kunne ha hatt denne i en env variabel. Tenker at det er greit at det står slik, endrer bare variabelnavnet til `bucketId` slik at det faktisk er litt tydeligere hva det er.